### DISCUSSION TITLE
Sprint 1 Retrospective

### DISCUSSION THREAD

**Discussion by coffee-ns** [coffee-ns/hotjava#14] [General]
Everyone, drop your comments for your sprint 1 retrospective here!

**Comment by mcquadmj117** [comment]
Sprint 1 
**What We Accomplished:**

- Set up a plan for communicating (how and when) which has worked well so far
- Set up a plan for what we each of else wanted to work on, and what we needed to work on
- Implemented the skeleton/backbone of our app with dto/dao/services and controllers set up and ready to have functionality added
- created the first view/pages of our application and got thymeleaf set up for passing in data and making reusable components
- created basic tests using mocked daos and were able to run them successfully with appropriate changes to the services and controllers

**TODOs from sprint 1 that we will need to carry over to next sprint and backlog items:**

- Fix yaml for circle ci and test it
- Finish setting up test methods as more functionality is give to app
- Begin implementing how we will show notifications/confirmations to the user 
- Forms functionality
- Java docs and warning fixes

**Comment by McMas1999** [comment]
Sprint 1
What We Accomplished:

- Created the foundation of our app. 
- Created Homepage that is built with Bootstrap.
- Created Mock DAOs
- Created basic tests.
- Implemented Fragments.

What we still need to accomplish: 
- A better method of assigning work. 
- Peer Review coding.
- Circle CI
- Proper Documentation

**Comment by zacharba** [comment]
Sprint 1
**What We Accomplished**

- Setting up the frame for our application.
- Created and Initialized pages and planned for upcoming expectations for those pages.
- Reduced Code clutter and initialized a library for repeated code using Thymeleaf fragments.
- Created Mock DAO's and Interfaces
- Built tests that implement the application logic

**Sprint 2 Expectations**

- Linear workflow with accomplishable and measurable tasks for each member
- Circle CI testing was running into a few issues, want to establish and get that working before moving forward
- Applying logic and outline to application pages
- Building new tests and logic
- Apply good coding standards and continue to utilize fragments for HTML
